test/yapi_network_test.rb

```diff
@@ -0,0 +1,374 @@
+#! /usr/bin/env rspec
+
+require_relative "test_helper"
+
+require "yast"
+
+Yast.import "YaPI::NETWORK"
+Yast.import "NetworkInterfaces"
+Yast.import "Routing"
+Yast.import "IP"
+Yast.import "Service"
+Yast.import "DNS"
+Yast.import "LanItems"
+
+describe Yast::YaPI::NETWORK do
+  include YaPINetworkStub
+
+  describe ".Write" do
+
+    context "with routes" do
+      # Mock the system to do not break anything
+      before do
+        allow(Yast::Routing).to receive(:Read)
```

Comment:
In this case, is `Yast::Routing` the subject? I though it was `Yast::YaPI::NETWORK`.